library/src/main/java/com/cube/storm/content/lib/helper/BundleHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.cube.storm.content.lib.helper;
+
+import android.net.Uri;
+import com.cube.storm.ContentSettings;
+import com.cube.storm.content.model.Manifest;
+
+/**
+ * Utility methods relating to Storm content bundles and their contents
+ */
+public class BundleHelper
+{
+	public static Long readContentTimestamp()
+	{
+		Uri manifestUri = Uri.parse("cache://manifest.json");
+		Manifest manifest = ContentSettings.getInstance().getBundleBuilder().buildManifest(manifestUri);
+
+		if (manifest == null)
+		{
+			return null;
+		}
+
+		return manifest.getTimestamp();
```

Comment:
Hi @timxyz , could it be simplified with `return manifest == null ? null : manifest.getTimeStamp();` ?